Dockerfile

```diff
@@ -33,18 +33,17 @@ RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/nul
 # install gcc-c++ make
 RUN apt-get -y install build-essential
 
-ENV MKLROOT=/opt/intel/mkl/lib/intel64/
+ENV MKLROOT=/opt/intel/mkl
 
 # build faiss and the c api
 RUN apt-get -y install git && \
     git clone -b ${FAISS_VERSION} https://github.com/facebookresearch/faiss.git /faiss && \
     cd /faiss && \
-    cmake -DFAISS_ENABLE_GPU=OFF -DFAISS_ENABLE_PYTHON=OFF -DBUILD_TESTING=ON -DCMAKE_BUILD_TYPE=Release -DFAISS_ENABLE_C_API=ON -DBUILD_SHARED_LIBS=ON -DFAISS_OPT_LEVEL=avx2 -B build . && \
-    make -C build -j $(nproc) faiss && \
-    make -j $(nproc) -C build install && \
-    cd /
+    git checkout add3705c1147a51b1b20161cc6c08945bd485f14 && \
```

Comment:
What is the significance of this SHA? Is there a tag for it that we could use instead?

Reply:
i'll try using their latest release tag and see if the speedup improvements still work. agree it would be better to use a release tag.